### DISCUSSION TITLE
Update Range Slider min/max values? (Vue)

### DISCUSSION THREAD

**Discussion by samydoesit** [chakra-ui/zag#945] [General]
Hello great project :)
I have a question is there a way to dynamically update the min/max values of the range-slider machine using Vue SFC's?

**Comment by segunadebayo** [comment]
Hi @samydoesit,

You should be able to use the transient updates as described here:
https://zagjs.com/overview/programmatic-control#transient-updates

 `useMachine(slider.machine(...), { context: { min, max } })`

**Comment by segunadebayo** [comment]
Closing as outdated — please refer to the updated v1.0 API. Props can be updated dynamically via the framework adapters.
